k8s/upgrades/README.md

```diff
@@ -66,6 +68,16 @@ machine that has access to Kubernetes master**
   Verify that `apiserver` is listed. If you have installed with helm charts, 
   the apiserver name may be openebs-apiserver. 
 
+  **From 1.4.0 onwards the webhook service, secret and validatingwebhookconfiguration 
+  will be managed by the admission-server once it starts. This requires removal of old 
+  configurations using below commands :**
+
+  ```sh
+  kubectl delete svc admission-server-svc -n openebs
+  kubectl delete secret admission-server-certs  -n openebs
+  kubectl delete validatingwebhookconfiguration openebs-validation-webhook-cfg
```

Comment:
@shubham14bajpai  validatingwebhookconfiguration name differs in operator and helm based deployments.

Reply:
Added a note for the same.